extract/bbox_merge.py

```diff
@@ -0,0 +1,18 @@
+from .coordinate import lower_right_most_coordinate, upper_left_most_coordinate
+from .bbox import BBox
+
+
+def bbox_merge(*bboxes):
```

Comment:
I don't *think* you need the `*bboxes` here; it's a useful construct when you don't know how many things you're being passed, but in this case you do know what you're being passed - exactly one object, which is a list, which contains zero or more bboxes